src/components/BotBuilder/BotBuilder.js

```diff
@@ -69,7 +84,37 @@ class BotBuilder extends React.Component {
                 <div style={styles.home} className="botbuilder-page-wrapper">
                     <Paper style={styles.paperStyle} className="botBuilder-page-card" zDepth={1}>
                         <Grid>
-                            <Row>
+                            <Row style={(!this.state.createBotWizard)?({display: 'none'}):({display: 'block'})}>
+                            <div style={{textAlign: 'center'}}>
+                                <RaisedButton
+                                    label='Use pre-coded Contact Bot'
+                                    backgroundColor={colors.header}
+                                    labelColor='#fff'
+                                    onClick={this.showContactCode}
+                                />
+                            </div>
+                            <div className={'code-wrap '+(this.state.contactCode?'show':'hide')}>
+                                <br />
+                                <div className="code-box">
+                                    <code>
+                                        &lt;script type=&quot;text/javascript&quot; id=&quot;susi-bot-script&quot; data-token=&quot;{cookies.get('loggedIn')}&quot; src=&quot;https://skills.susi.ai/susi-chatbot.js&quot;&gt;&lt;/script&gt;
```

Comment:
@DNS-404 I think it should be `susi-contactbot.js` instead of `susi-chatbot.js` here.